app/controllers/application_controller.rb

```diff
@@ -1,7 +1,6 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :configure_permitted_parameters, if: :devise_controller?
-  before_action :authenticate_user!
```

Comment:
Why we remove it from here? don't we want all controllers/actions to be hidden from unauthorized users?